trieste/utils/pareto.py

```diff
@@ -105,3 +105,32 @@ def _bounds_2d(front: TensorType) -> BoundedVolumes:
         upper = tf.concat([range_ + 1, pf_ext_idx[::-1, 1:][: pf_ext_idx[-1, 0]]], axis=-1)
 
         return BoundedVolumes(lower, upper)
+
+    def hypervolume_indicator(self, reference: tf.Tensor) -> tf.Tensor:
+        """
+        Method to calculate the hypervolume indicator
+        The hypervolume indicator is the volume of the dominated region.
+
+        :param reference: a reference point to use, with shape [D].
```

Comment:
Can you expand on what a reference point is? From the perspective of someone who doesn't know much about multi-objective, or is used to a different name

Reply:
this term is specific for hypervolume-based Pareto optimization. mathematically reference point = max(Pareto front set)+epsilon. epsilon is some constant. visually it is explained in Figure 2 of this article https://link.springer.com/content/pdf/10.1007/s10898-013-0118-2.pdf . 
not sure what to add In the docstring. what if "a bound point to calculate the volume"

Reply:
how about "Defines the upper bound of the hypervolume"?

Reply:
sounds good